### DISCUSSION TITLE
Reaching Out To Mentors  For Self Proposed GSOC Project

### DISCUSSION THREAD

**Discussion by mahajanparth** [openvinotoolkit/openvino#23502] [Google Summer of Code]
Hello OpenVino Mentors !!! 

I am Parth Mahajan, currently working as a Research Assistant at IIIT-Hyderabad, my interest lies in developing 3D slam pipelines and motion planning for robots and I believe acceleration platforms make a lot of sense in robotics. I see a lot of potential in using openvino frame work to accelerate robotics pipelines and hence would like to contribute to openvino from that perspective. I want to propose a self-project for GSOC 2024 and would like to know if it is possible. I am passionate about building a practical product and guidance from the current team would be a good alignment to the same.
Main Idea in Rough :
1)Accelerating Feature Correspondence b/w different frames using Super Glue or similar Architecture using Openvino
2) Using the above information to get the Rt matrix to update the factor graph. Look for loop closure in the information matrix
3) Make Modules for Slam Pipelines
Would love to discuss the same with the team !!!
May I know if it is possible to propose a self-project for GSoc ?

Meanwhile i have already started contributing to the repo at  PR : [#20922](https://github.com/openvinotoolkit/openvino/issues/20922) , [#21986](https://github.com/openvinotoolkit/openvino/issues/21986) , [#21986](https://github.com/openvinotoolkit/openvino/issues/21986)

**Comment by adrianboguszewski** [comment]
@mlukasze who would be interested in this project?

**Reply by mahajanparth** [reply]
Hello, May I know if anyone is interested ?
